lib/init.php

```diff
@@ -57,3 +57,27 @@ function gutenberg_menu() {
 	);
 }
 add_action( 'admin_menu', 'gutenberg_menu', 9 );
+
+
+/**
+ * This should be implemented in core as a default
+ * post type feature.
+ *
+ */
+if ( ! function_exists( 'add_content_support' ) ) {
+	/**
+	 * Add the 'content' support to core content post types.
+	 *
+	 * @since 6.8.0
+	 */
+	function add_content_support() {
+		global $wp_post_types;
+
+		foreach ( array_keys( $wp_post_types ) as $post_type ) {
+			if ( in_array( $post_type, array( 'post', 'page', 'wp_template' ), true ) ) {
```

Comment:
It seems odd that `wp_template` has a post type support called `content`. 

In all the other places where we differentiate between wether something is content or now `wp_template` explicitly counts as non content

Reply:
Yes, the support type name is debatable b/c of this. It's also still unclear what is content and what is not ... Let's see what other reviewers think.

Reply:
What should the default be for custom post types that plugins might be creating? I.e. is the intention that we're gating the zoom out feature so that plugins will need to opt-in in order to support it?

(I don't have an opinion either way, just a thought as I looked over the PR 🙂)

Reply:
There are some kinds of post types where it doesn't make sense. We can make all post types support this by default and opt out only when a custom supports list is provided.